src/main/webapp/help-retentionPolicyId.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    (Optional) The ID of the retention policy for the VDB.
```

Comment:
Capitalize `Retention` since it's one of our main Policy Types.

https://cd.delphix.com/docs/latest/policies-for-scheduled-jobs